lib/TraackrApi/Analysis.php

```diff
@@ -4,23 +4,29 @@
 
 class Analysis extends TraackrApiObject {
 
-   public static function toplinks($p = array('min_linkbacks' => 10, 'count' => 5)) {
-
-      $analysis = new Analysis();
-      $p = $analysis->addCustomerKey($p);
-
-      // support for multi params
-      if ( isset($p['influencers']) ) {
-         $p['influencers'] = is_array($p['influencers']) ?
-            implode(',', $p['influencers']) : $p['influencers'];
-      }
-      if ( isset($p['tags']) ) {
-         $p['tags'] = is_array($p['tags']) ?
-            implode(',', $p['tags']) : $p['tags'];
-      }
-
-      return $analysis->get(TraackrApi::$apiBaseUrl.'analysis/toplinks', $p);
-
-   } // End function toplinks()
-
-} // End class Analysis
\ No newline at end of file
+    public static function toplinks($p = array('min_linkbacks' => 10, 'count' => 5))
+    {
+        $analysis = new Analysis();
+        $p = $analysis->addCustomerKey($p);
+
+        // support for multi params
+        if ( isset($p['influencers']) ) {
+            $p['influencers'] = is_array($p['influencers']) ?
+                implode(',', $p['influencers']) : $p['influencers'];
+        }
+        if ( isset($p['tags']) ) {
+            $p['tags'] = is_array($p['tags']) ?
+                implode(',', $p['tags']) : $p['tags'];
+        }
+
+        return $analysis->get(TraackrApi::$apiBaseUrl.'analysis/toplinks', $p);
+    }
+
+    public static function keywords($p = array())
+    {
+        $analysis = new Analysis();
+        $p = $analysis->addCustomerKey($p);
```

Comment:
what's this for? the analysis endpoint doesn't need a customer key, is this for something else on the PHP side?

Reply:
Ah, thanks! Yes, that's not necessary and the reason it wasn't bombing for me is because that function only adds a customer key into the response if one has been set globally. I'll remove it; you're right, it's unnecessary for this call.